Userland/Games/Solitaire/Game.h

```diff
@@ -140,6 +154,9 @@ class Game final : public GUI::Frame {
     }
 
     void mark_intersecting_stacks_dirty(Card& intersecting_card);
+    void score_move(CardStack& from, CardStack& to, bool inverse);
+    void undo_save_move(CardStack& from, CardStack& to, NonnullRefPtrVector<Card> moved_cards);
+    void undo_save_flip(Card& card);
```

Comment:
```suggestion
    void save_undo_move(CardStack& from, CardStack& to, NonnullRefPtrVector<Card> moved_cards);
    void save_undo_flip(Card& card);
```

This is a bit more subjective, but these method names sounded kinda awkward to me.

Reply:
Yeah I agree. I thought about it more and updated it to be `remember_move_for_undo` and `remember_flip_for_undo` which I think are clearer.